build/build-locale.ts

```diff
@@ -3,12 +3,19 @@ import fs from 'fs'
 import save from 'file-save'
 import { resolve, basename } from 'path'
 import { buildOutput } from './paths'
+import babel from '@babel/core'
+
+import type { FileResultCallback } from '@babel/core'
 
 const localePath = resolve(__dirname, '../packages/locale/lang')
 const fileList = fs.readdirSync(localePath)
 
-const transform = function (filename, name, cb) {
-  require('@babel/core').transformFile(
+const transform = function (
+  filename: string,
+  name: string,
+  cb: FileResultCallback
+) {
+  babel.transformFile(
```

Comment:
In fact this file is already deprecated right now since we applied `gulp` for building locale code. (But this change does seem legit.

Reply:
So could I remove `build-locale.ts` and `yarn build:locale-umd`?